.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,3 +17,14 @@
 * [ ] Command line build passes?
 * [ ] Has this been smoke tested?
 * [ ] Testing instructions included in associated issue?
+
+### Reviewer Checklist
+
+* [ ] Changes appear to address issue?
+* [ ] Changes appear not to be breaking changes?
+* [ ] Appropriate unit tests included?
+* [ ] Code style and in-line documentation are appropriate?
+* [ ] Commit messages meet standards?
+* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
+* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)
+* [ ] List of Acceptance Tests Performed. These will be turned into e2e tests.
```

Comment:
Could we remove this for now just to get the automation stuff in? The specifics of per-PR testing are currently under review.